SamplesDashboard/SamplesDashboard/ClientApp/src/views/details/Details.tsx

```diff
@@ -88,48 +92,56 @@ export default class Details extends React.Component<any, any> {
         // call the statistics function
         this.statusStatistics(this.allItems);
         this.setState({
-            items: this.allItems,
+            items: this.sortItems(this.allItems),
             repositoryDetails: {
-                name: repositoryName,
-                description: data.description,
+            name: repositoryName,
+            description: data.description,
                 url: data.url
             },
             isLoading: false
         });
     }
 
-    // compute status statistics 
+    public sortItems(items: IDetailsItem[]) {
+        const sortedItems = items.sort((a, b) => (a.packageName.toLowerCase() > b.packageName.toLowerCase()) ? 1 : -1);
+        return sortedItems;
+    }
+    // compute status statistics
     public statusStatistics(items: IDetailsItem[]) {
         let uptoDateCount = 0;
         let urgentUpdateCount = 0;
         let majorUpdateCount = 0;
         let patchUpdateCount = 0;
         let unknownCount = 0;
```

Comment:
A type can be created to help define what goes in to the statistics like
`export interface IUpdateStats {
  count: number;
  percent: number;
  total: number;
}`

So that we can then use it like so
```suggestion
  let uptoDate :IUpdateStats = {
    count: 0,
    percent: 0,
    total: 0
  }
  let urgentUpdate :IUpdateStats = {
    count: 0,
    percent: 0,
    total: 0,
  }
  let majorUpdate :IUpdateStats = {
    count: 0,
    percent: 0,
    total: 0,
  }
  let patchUpdate :IUpdateStats = {
    count: 0,
    percent: 0,
    total: 0
  }
  let unknown :IUpdateStats = {
    count: 0,
    percent: 0,
    total: 0
  }
```

Thus we eliminate the need to have related variables that are getting declared separately